### PR TITLE
Add nodejs and npm

### PR DESCRIPTION
**Pull request by RothAndrew**

### CODE REVIEW

**Comment by RothAndrew** [comment]
@phumberdroz can you give this a quick glance?

File: src/docker/Dockerfile [hunk 1]
```diff
@@ -96,6 +96,11 @@ RUN curl -sL https://packages.microsoft.com/keys/microsoft.asc | \
     tee -a /etc/apt/sources.list.d/azure-cli.list && \
     install_packages azure-cli=${AZURE_CLI}
 
+# Install Nodejs and NPM
+ARG NODEJS_VERSION=10.15.2~dfsg-2
```
Comment (phumberdroz) [inline src/docker/Dockerfile]: this is not the current LTS it is 12.10 i believe

Comment (RothAndrew) [reply src/docker/Dockerfile]: It is the latest version in the debian buster package library, but you make a good point. I will look into installing from another source, perhaps a different PPA

Comment (RothAndrew) [reply src/docker/Dockerfile]: Actually it may be a better idea to install [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md)

Comment (phumberdroz) [reply src/docker/Dockerfile]: AFAIK and why I did not do this yet is that node uses multipile PPA for each of their releases streams. Current, LTS and still maintained maintance streams are each seperate.

NVM seems to be a valid approach.

**Comment by RothAndrew** [comment]
@phumberdroz can you take another look?

File: src/docker/Dockerfile [hunk 2]
```diff
@@ -63,42 +63,52 @@ RUN curl -LO https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9
...
+  echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ buster main" | \
+  tee -a /etc/apt/sources.list.d/azure-cli.list && \
+  install_packages azure-cli=${AZURE_CLI}
 
 # Quit being root for safety.
 USER dadsgarage
 
+# Install Node Version Manager, NodeJS, and NPM
+ARG NVM_VERSION=0.35.2
+ARG NODE_VERSION=node
```
Comment (phumberdroz) [inline src/docker/Dockerfile:104]: which node version gets installed can we make this more explicit?

**Review by phumberdroz — APPROVED** [review]
Other than the comment I left LGTM 👍

File: src/docker/Dockerfile (hunk 2 above)
Comment (RothAndrew) [reply src/docker/Dockerfile:104]: `node` is an alias NVM uses for the latest version. I can change it to pin the version.

Comment (phumberdroz) [reply src/docker/Dockerfile:104]: Considering that all other applications are pinned at a specific version I think the same should apply here and the default should be the current LTS with 12.15 being the current LTS version.

**Pull request closed without merging**